docs/notebooks/failure_ego.pct.py

```diff
@@ -100,49 +106,78 @@ def observer(x):
         FAILURE: trieste.data.Dataset(x, tf.cast(np.isfinite(y), tf.float64))
     }
 
+
 # %% [markdown]
 # We can evaluate the observer at points sampled from the search space.
 
 # %%
 num_init_points = 15
 initial_data = observer(search_space.sample(num_init_points))
 
+
 # %% [markdown]
-# ## Modelling the data
+# ## Model the data
 #
-# We'll model the data on the objective with a regression model, and the data on which points failed with a classification model. The regression model will be a `GaussianProcessRegression` wrapping a GPflow `GPR`, and the classification model a `VariationalGaussianProcess` wrapping a GPflow `VGP` with Bernoulli likelihood. We'll train both models with L-BFGS-based optimizers.
+# We'll model the data on the objective with a regression model, and the data on which points failed with a classification model. The regression model will be a `GaussianProcessRegression` wrapping a GPflow `GPR`, and the classification model a `VariationalGaussianProcess` wrapping a GPflow `VGP` with Bernoulli likelihood.
 
 # %%
 def create_regression_model(data):
     variance = tf.math.reduce_variance(data.observations)
-    kernel = gpflow.kernels.Matern52(variance=variance, lengthscales=0.2 * np.ones(2,))
+    kernel = gpflow.kernels.Matern52(variance=variance, lengthscales=0.2 * np.ones(2, ))
     gpr = gpflow.models.GPR(astuple(data), kernel, noise_variance=1e-5)
     set_trainable(gpr.likelihood, False)
     return gpr
 
 
 def create_classification_model(data):
-    variance = tf.math.reduce_variance(data.observations)
-    kernel = gpflow.kernels.SquaredExponential(variance, lengthscales=0.1 * np.ones(2,))
+    kernel = gpflow.kernels.SquaredExponential(variance=100., lengthscales=0.2 * np.ones(2, ))
     likelihood = gpflow.likelihoods.Bernoulli()
     vgp = gpflow.models.VGP(astuple(data), kernel, likelihood)
+    set_trainable(vgp.kernel.variance, False)
     return vgp
 
 
 regression_model = create_regression_model(initial_data[OBJECTIVE])
 classification_model = create_classification_model(initial_data[FAILURE])
 
+# %% [markdown]
+# ## Create a custom optimize method
```

Comment:
do we need a new section?

Reply:
I felt it would be useful if we want "how-to" pointers